src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -1133,7 +1136,11 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
 
       "should 204 if everything works" in {
         arrowServer
-          .when(request().withMethod("POST").withPath("/avroToRawls"))
+          .when(request().withMethod("POST").withPath("/avroToRawls").withHeaders(
+            Header.header("Accept-Encoding", "gzip"),
+            Header.header(NottableString.not("x-firecloud-id"), NottableString.string(".*")),
+            Header.header(NottableString.not("authorization"), NottableString.string(".*"))
```

Comment:
Yes, it's ugly, but `header` would conflict with spray and `not` would conflict with scalatest (I think that's what it was... there was definitely some conflict).